src/Symfony/Component/Notifier/Bridge/Firebase/FirebaseOptions.php

```diff
@@ -27,10 +27,13 @@ abstract class FirebaseOptions implements MessageOptionsInterface
      */
     protected $options;
 
-    public function __construct(string $to, array $options)
+    protected $data;
```

Comment:
should be private to follow our coding standard

Reply:
You are right, but as explained I won't go with a dedicated property here, as far as I understood the problem 🤗